src/stream/src/executor/dispatch.rs

```diff
@@ -225,6 +234,14 @@ impl DispatchExecutorInner {
                     }
                 }
             }
+            Mutation::Combined(v) => match &v[..] {
+                [Mutation::Add { .. }, Mutation::Update { .. }] => {
```

Comment:
As there's only a single form of combination here, what about directly naming it as `AddAndUpdate`? 🤣 Then it can also be type-safer.

This requires us to extract the fields of `Mutation::Add` and `Mutation::Update` to dedicated structs.